aws-cli-assumerole.sh

```diff
@@ -1,28 +1,47 @@
 #! /bin/bash
 #
 # Dependencies:
-#   brew install jq
+#   MacOS : brew install jq
+#   Linux (deb based) : apt install jq
 #
 # Setup:
 #   chmod +x ./aws-cli-assumerole.sh
 #
 # Execute:
-#   source ./aws-cli-assumerole.sh
+#   source ./aws-cli-assumerole.sh <AWS_ID> [<AWS_ROLE> [<SESSION_NAME>]]
 #
 # Description:
 #   Makes assuming an AWS IAM role (+ exporting new temp keys) easier
 
-unset  AWS_SESSION_TOKEN
-export AWS_ACCESS_KEY_ID=<user_access_key>
-export AWS_SECRET_ACCESS_KEY=<user_secret_key>
-export AWS_REGION=eu-west-1
+AWS_PROFILE="default"
+AWS_ROLE="OrganizationAccountAccessRole"
```

Comment:
I don't know why I didn't use `~/.aws/credentials` originally? 🤦‍♂️ 

I imagine I was probably being lazy and not wanting to have to parse the creds from the file 🤷‍♂️

👍 to this change regardless